apps/home/templates/apps_home/blocks/paragraph_block.html

```diff
@@ -0,0 +1,7 @@
+{% load block_tags %}
+
+<div class="modul-servicepanel fullwidth ds-block ds-block--body-bg">
```

Comment:
service panel and full width we used for elements that needed to have the coloured background over the full width of the page, as we no longer need that, I would now use their `modul-text_bild` just without a bild like they do in their article example page

Reply:
So, I replace the classes that are there now with `modul-text_bild ds-block ds-block--body-bg`, right? Will do!